articles/files_and_versioning.md

```diff
@@ -165,9 +164,7 @@ file = syn.store(file, forceVersion=False)
 # Get file from Synapse, set download=False since we are only updating annotations
```

Comment:
don't actually have to get the file at all - `synSetAnnotations` does a store command (though I don't like it).

Reply:
do whatever is consistent w/ Python. I still don't like it.